src/lib/moduleMocker.js

```diff
@@ -47,6 +47,48 @@ function getType(ref) {
   return null;
 }
 
+function PlainConstructor() {}
+var plainObject = new PlainConstructor();
+/**
+ * methods on ES6 classes are not enumerable, so they can't
+ * be found with a simple `for (var slot in ...) {`
+ *
+ * scan will return an object with the same shape you would
+ * get with `for ... in` except the non-enumerable methods
+ * are included.
+ */
+ function scan(obj) {
+   var fakeObj = {};
+   var prop;
+   for (prop in obj) {
+     fakeObj[prop] = true;
+   }
+   if (obj) {
+     var props = [];
+     var tmp = obj;
+     var descriptor;
+
+     do {
+       props = Object.getOwnPropertyNames(tmp);
+
+       for (var i = 0; i < props.length; i++) {
+         prop = props[i];
+         // Skip methods that exist on all objects.
+         if (!plainObject[prop]) {
```

Comment:
so if you have

```
class A {
  toString() {}
}
```

Will the check above exclude the `toString` of the `A` class from mocking?

I think we just need to blacklist the properties we don't wanna mock (like `constructor`, and similar).

I meant not to consider `Object.prototype` in this traversal:

```
...
tmp = Object.getPrototypeOf(tmp);
} while (tmp !== Object.prototype);
```

Unless we need to mock the things from `Object.prototype` as well, but previous `for-in` loop wouldn't get anything from `Object.prototype` because by default the things aren't enumerable there.